app/Console/Commands/ReportDevices.php

```diff
@@ -21,8 +22,13 @@ public function __construct()
         parent::__construct();
         $this->addArgument('device spec', InputArgument::OPTIONAL);
         $this->addOption('fields', 'f', InputOption::VALUE_REQUIRED, default: 'hostname,ip');
-        $this->addOption('output', 'o', InputOption::VALUE_REQUIRED, __('commands.report:devices.options.output', ['types' => '[table, csv, none]']), 'table');
+        $this->addOption('output', 'o', InputOption::VALUE_REQUIRED, __('commands.report:devices.options.output', ['types' => '[table, csv, json, none]']), 'table');
         $this->addOption('list-fields');
+        $this->addOption('list-relationships', '-L', InputOption::VALUE_NONE);
+        $this->addOption('no-header', 't', InputOption::VALUE_NONE);
+        $this->addOption('devices-as-array', 'a', InputOption::VALUE_NONE);
+        $this->addOption('relationships', 'r', InputOption::VALUE_OPTIONAL);
+        $this->addOption('all-relationships', 'R', InputOption::VALUE_NONE);
```

Comment:
This option isn't needed, you can just do all.  Also, all seems crazy.

Reply:
It is handy for quickly seeing what is present. Having to include that all by hand would be annoying. Also this data gets big quickly so having the option of not including everything is handy. Allows one to only fetch what they want.